src/NUnitEngine/nunit.engine.tests/Drivers/NUnit3FrameworkDriverTests.cs

```diff
@@ -23,10 +23,10 @@
 
 using System;
 using System.Collections.Generic;
-using System.Web.UI;
+using System.Reflection;
 using System.Xml;
 using NUnit.Tests.Assemblies;
-using NUnit.Engine.Internal;
+using NUnit.Engine.Internal.Tests;
```

Comment:
Why this change? What are you referring to? The CI is failing because of an ambiguous reference to Assert, between NUnit.Framework.Assert and NUnit.Engine.Internal.Tests.Assert, which I can't see anywhere.